src/main/webapp/app/course/tutorial-groups/tutorial-groups-instructor-view/tutorial-groups/tutorial-groups-management/tutorial-groups-management.component.html

```diff
@@ -10,77 +10,14 @@ <h4>{{ 'artemisApp.pages.tutorialGroupsManagement.title' | artemisTranslate }}</
-                            <th jhiSortBy="isOnline">
-                                <a class="th-link">{{ 'artemisApp.entities.tutorialGroup.isOnline' | artemisTranslate }}</a>
-                                <fa-icon [icon]="faSort"></fa-icon>
-                            </th>
-                            <th jhiSortBy="language">
-                                <a class="th-link">{{ 'artemisApp.entities.tutorialGroup.language' | artemisTranslate }}</a>
-                                <fa-icon [icon]="faSort"></fa-icon>
-                            </th>
-                            <th jhiSortBy="campus">
-                                <a class="th-link">{{ 'artemisApp.entities.tutorialGroup.campus' | artemisTranslate }}</a>
-                                <fa-icon [icon]="faSort"></fa-icon>
-                            </th>
-                            <th></th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr *ngFor="let tutorialGroup of tutorialGroups; trackBy: trackId" id="tutorial-group-row-{{ tutorialGroup.id }}">
-                            <td>
-                                <a [routerLink]="['/course-management', courseId, 'tutorial-groups-management', tutorialGroup.id]">{{ tutorialGroup.id }}</a>
-                            </td>
-                            <td>
-                                <span class="d-inline-block text-truncate title"> {{ tutorialGroup.title }} </span>
-                            </td>
-                            <td>{{ tutorialGroup.teachingAssistant?.name }}</td>
-                            <td>{{ tutorialGroup.capacity }}</td>
-                            <td>{{ tutorialGroup.registrations?.length }}</td>
-                            <td>
-                                {{ (tutorialGroup.isOnline ? 'artemisApp.generic.online' : 'artemisApp.generic.offline') | artemisTranslate }}
-                            </td>
-                            <td>
-                                {{ (tutorialGroup.language === GERMAN ? 'artemisApp.generic.german' : 'artemisApp.generic.english') | artemisTranslate }}
-                            </td>
-                            <td>
-                                <span class="d-inline-block text-truncate long-text"> {{ tutorialGroup.campus }} </span>
-                            </td>
-                            <td>
-                                <jhi-tutorial-group-row-buttons
-                                    [courseId]="courseId"
-                                    [tutorialGroup]="tutorialGroup"
-                                    (tutorialGroupDeleted)="loadTutorialGroups()"
-                                ></jhi-tutorial-group-row-buttons>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
+            <ng-container *ngIf="tutorialGroups && tutorialGroups.length > 0">
```

Comment:
Please test, this should work too. 🙂 
```suggestion
            <ng-container *ngIf="tutorialGroups?.length">
```

Reply:
It works! Changed it in 404a61f9e625e0e40fb810485d800baaea28b685